.circleci/config.yml

```diff
@@ -23,6 +23,7 @@ orbs:
             name: Install basic packages
             command: |
               sudo killall -9 apt-get || true && \
+              sudo apt-get update && \
```

Comment:
It's already called in `install_erlang` command.

Reply:
If you were missing that call in some job, simply change the order of `fetch_packages` and `install_erlang` commands